### PR TITLE
Add devstack installation instruction

### PR DESCRIPTION
**Pull request by Xenwar**
Add devstack installation instruction

Signed-off-by: Anwar Hassen <anwar.hassen@est.tech>

### CODE REVIEW

File: capo/README.md [hunk 1]
```diff
@@ -27,6 +27,68 @@ Then:
 * Run `make provision` to deploy a 2 node cluster on OpenStack and test your
   changes
 
+## Devstack setup
+
+In order to add more features, such as trunking, we need to create an Openstack instance using devstack.
+
+1. Create Centos-8 VM with 32GB or RAM and 8 CPU cores.
```
Comment (wgslr) [inline capo/README.md:34]:
```suggestion
1. Create Centos-8 VM with 32GB of RAM and 8 CPU cores.
```

File: capo/README.md [hunk 2]
```diff
@@ -27,6 +27,68 @@ Then:
...
+Now, start the installation.
+
+```bash
+./stack.sh
+```
+
+Verify installation
+
+```bash
+openstack extension list --network | grep -i trunk
```
Comment (macaptain) [inline capo/README.md:89]: What's the output when you run this on Devstack?

**Review by macaptain — DISMISSED** [review]

File: capo/README.md [hunk 3]
```diff
@@ -27,6 +27,68 @@ Then:
...
+
+1. Create Centos-8 VM with 32GB or RAM and 8 CPU cores.
+2. From within the VM, clone the devstack repo
+
+```bash
+git clone https://opendev.org/openstack/devstack.git
+```
+3. Create stack user/group with the proper priviledge
+```bash
+sudo ./tools/create-stack-user.sh
```
Comment (wgslr) [inline capo/README.md:42]:
```suggestion
```bash
cd devstack
sudo ./tools/create-stack-user.sh
```

File: capo/README.md [hunk 4]
```diff
@@ -27,6 +27,68 @@ Then:
...
+```bash
+sudo su
+su - stack
+```
+
+```bash
+git clone https://opendev.org/openstack/devstack.git
+cd devstack
+sudo mkdir -p /opt/stack/logs
+sudo chown stack.stack /opt/stack/
```
Comment (wgslr) [inline capo/README.md:55]:
```suggestion
sudo chown stack:stack /opt/stack/
```

File: capo/README.md [hunk 5]
```diff
@@ -27,6 +27,68 @@ Then:
...
+If you see the above error, run the following to fix it.
+
+```bash
+sudo sed -i 's/enabled=1/enabled=0/' /etc/yum.repos.d/rdo-release.repo
+yum update -y
+```
+
+configure trunk support, more information can be found [here](https://docs.openstack.org/ocata/networking-guide/config-trunking.html)
+
+To add trunk supoort edit ```lib/neutron_plugins/ovn_agent```
```
Comment (wgslr) [inline capo/README.md:74]:
```suggestion
To add trunk supoort edit `lib/neutron_plugins/ovn_agent`
```

File: capo/README.md (hunk 2 above)
Comment (Xenwar) [reply capo/README.md]: Thanks, added a different command.

File: capo/README.md (hunk 4 above)
Comment (Xenwar) [reply capo/README.md:55]: changed. both have the same effect.

Comment (wgslr) [reply capo/README.md]: Oh nice! I didn't know about the dot syntax

**Review by macaptain — APPROVED** [review]

File: capo/README.md [hunk 6]
```diff
@@ -27,6 +27,72 @@ Then:
...
+4. Install devstack as stack user
+
+```bash
+sudo su
+su - stack
+```
+
+```bash
+git clone https://opendev.org/openstack/devstack.git
+cd devstack
```
Comment (furkatgofurov7) [inline capo/README.md:54]: why do we need these, we already have them in steps 2 and 3?

File: capo/README.md [hunk 7]
```diff
@@ -27,6 +27,72 @@ Then:
...
+Error: Failed to download metadata for repo 'openstack-wallaby': Cannot prepare internal mirrorlist: No URLs in mirrorlist
+```
+If you see the above error, run the following to fix it.
+
+```bash
+sudo sed -i 's/enabled=1/enabled=0/' /etc/yum.repos.d/rdo-release.repo
+yum update -y
+```
+
+configure trunk support, more information can be found [here](https://docs.openstack.org/ocata/networking-guide/config-trunking.html)
```
Comment (furkatgofurov7) [inline capo/README.md:73]: Is this is a new paragraph? at least should be started with capital C.

File: capo/README.md [hunk 8]
```diff
@@ -27,6 +27,72 @@ Then:
...
+
+To add trunk supoort edit `lib/neutron_plugins/ovn_agent`
```
Comment (furkatgofurov7) [inline capo/README.md:75]: typo: `support`

**Review by furkatgofurov7 — COMMENTED** [review]
Well, even though it is merged, some minor issues I spotted and could be improved, see below:

**Pull request merged**